Dockerfile

```diff
@@ -27,8 +27,8 @@ RUN opam init --disable-sandboxing -a --bare https://github.com/ocaml/opam-repos
 RUN opam switch create myswitch 4.14.2
 RUN opam exec -- opam install -y mirage opam-monorepo ocaml-solo5
 RUN opam exec -- opam install -y tls
-RUN opam pin https://github.com/palainp/miragevpn.git#update
-RUN opam pin https://github.com/palainp/mirage-qubes.git#update
+RUN opam pin https://github.com/robur-coop/miragevpn.git
+RUN opam pin https://github.com/mirage/mirage-qubes.git
```

Comment:
Once mirage-qubes is merged in opam-repository this line should be removed as it will be retrieved with `make depend`.
To me, if miragevpn could also be released in opam-repository, the previous line would also be removed :)

Reply:
As a side note, to improve reproducibility, I think it's better to add the commit hash if the package is not in opam-repository :)